src/libs/OptionsListUtils.js

```diff
@@ -464,6 +467,15 @@ function createOption(logins, personalDetails, report, reportActions = {}, {show
     }
 
     result.text = reportName;
```

Comment:
with the changes suggested i think this should updated to be, (feels simpler to read imo)
```
const parentReportActionMessage = ReportActionUtils::getParentReportAction(report);
result.text = _.isEmpty(parentReportActionMessage) ? reportName : parentReportActionMessage;
```